data/graphql/2.17/schema.docs-enterprise.graphql

```diff
@@ -2416,7 +2416,7 @@ type Commit implements GitObject & Node & Subscribable & UniformResourceLocatabl
   committedDate: DateTime!
 
   """
-  Check if commited via GitHub web UI.
+  Check if committed via GitHub web UI.
```

Comment:
Just adding a comment here, that these may be generated bugs from the product itself that need to be rebased out of this PR and addressed upstream

Reply:
Ooh, yeah, let me triage this for the ecosystem team so they can check out whether all of these can be merged 👍

Reply:
No problem, I think these were in the JSON files too, but I didn't touch those